apollo-runtime/src/main/java/com/apollographql/apollo/internal/interceptor/ApolloServerInterceptor.java

```diff
@@ -27,6 +27,9 @@
 import okhttp3.Response;
 import okio.Buffer;
 
+/**
+ * ApolloServerInterceptor is responsible for making the network calls to the server.
```

Comment:
We should probably add that this is the terminal interceptor (must be the last in order) as it doesn't call proceed on the chain.